step-cli/step-cli-launcher/src/main/java/step/cli/StepConsole.java

```diff
@@ -40,7 +44,7 @@
 
 @Command(name = "step",
         mixinStandardHelpOptions = true,
-        version = "step 1.0",
+        version = Constants.STEP_API_VERSION_STRING,
```

Comment:
We should define this centrally and use the same constant for the CLI and BE.

Reply:
@jeromecomte this constant is already in `step-core` (maybe not the best module to keep this, by the way) and it is reused in CLI and BE.